sw/device/tests/aon_timer_wakeup_irq_test.c

```diff
@@ -194,10 +194,15 @@ bool test_main(void) {
       &plic, kPlicTarget, kTopEarlgreyPlicIrqIdAonTimerAonWkupTimerExpired,
       kTopEarlgreyPlicIrqIdAonTimerAonWdogTimerBark);
 
-  // Executing the test using a randon time between 1 and 15 ms to make sure
+  // Executing the test using a randon time between 25 and 40 ms to make sure
```

Comment:
Could this value be calculated from the constants exposed in `sw/device/lib/arch/device.h`?